src/sentry/integrations/message_builder.py

```diff
@@ -46,8 +47,12 @@ def build_attachment_title(obj: Group | GroupEvent) -> str:
         group = getattr(obj, "group", obj)
         if group.issue_category == GroupCategory.PERFORMANCE:
             title = GROUP_TYPE_TO_TEXT.get(group.issue_type, "Issue")
+        elif hasattr(obj, "occurrence") and obj.occurrence is not None:
```

Comment:
can we condense this logic and re-use it (like line 96)?

Reply:
actually it looks like this gets repeated a few times, is it possible to put something in `BaseEvent` so we can make it one boolean to check

Reply:
or maybe even just switch the `hasattr` to `typeof(GroupEvent)`, feel like it reads easier